frontend/src/app/settings/modelsources/t4c-settings/edit-t4c-instance/edit-t4c-instance.component.html

```diff
@@ -176,9 +176,25 @@ <h2 *ngIf="!existing">Add a Team4Capella instance</h2>
           </button>
           <button mat-flat-button type="submit" color="primary">Submit</button>
         </div>
-        <div *ngIf="!editing && existing" class="flex justify-between">
-          <button mat-flat-button color="primary" (click)="enableEditing()">
-            Edit
+        <div
+          *ngIf="
+            !editing &&
+            existing &&
+            (t4cInstanceService.t4cInstance$ | async) !== undefined
+          "
+          class="flex justify-between"
+        >
+          <div *ngIf="!isArchived; else archivePlaceholder">
+            <button mat-flat-button color="primary" (click)="enableEditing()">
+              Edit
+            </button>
+          </div>
+          <ng-template #archivePlaceholder>
+            <div class="grow"></div>
+          </ng-template>
+          <button mat-flat-button color="primary" (click)="toggleArchive()">
+            <mat-icon class="mat-icon-position left">archive</mat-icon>
```

Comment:
Unarchive has the wrong symbol. It also shows the archive symbol, instead it should use the `unarchive` symbol.

Reply:
Done (didn't know that there is an `unarchive` symbol). Also has to be addressed in #1062